bootstrap/v2/pkg/kfapp/kustomize/kustomize.go

```diff
@@ -171,7 +171,7 @@ func GetKfApp(kfdef *kfdefsv2.KfDef) kftypes.KfApp {
 		},
 	}
 
-	// build restConfig and apiConfig using $HOME/.kube/config if the file exist
+	// build restConfig and apiConfig using $HOME/.kube/config if the file exists
```

Comment:
Please also update `Delete` method about deleteGlobalResources (I think we don't need this method any more since delete Application CR will delete GlobalResources, too)